lua/legendary/ui/format.lua

```diff
@@ -38,11 +38,15 @@ function M.default_format(item)
       item.description,
     }
   elseif Toolbox.is_function(item) then
+    -- stylua: ignore start
```

Comment:
Why is this here?

Reply:
Stylua makes lines in-between ugly otherwise

Reply:
I'd rather just have the formatter format it than stray from the style of everything else in the entire codebase.